.github/publish_docs.sh

```diff
@@ -2,21 +2,16 @@
 
 set -euv
 
-openssl aes-256-cbc -K $encrypted_None_key -iv $encrypted_None_iv -in .github/pulp-infra.enc -out ~/.ssh/pulp-infra -d
```

Comment:
I am just setting the public and private keys as secrets instead of storing them as `pulp-infra.enc`. Seems like storing them as secrets is just as secure as storing them as an encrypted file with the key/iv as a secret?

Reply:
Is it maybe some two-factor thing?
I know, the second factor (the encrypted file) is publicly lying around in the repository.